### DISCUSSION TITLE
How to customize the distance between Popover and the corresponding anchor point?

### DISCUSSION THREAD

**Discussion by cangSDARM** [rsuite/rsuite#2653] [Q&A] [answered]
The current distance between the Popover and the target anchor point is calculated in the rsuite (38px), which seems a bit too far in our own project. We want to shrink it into 30px, how to do it?

**Comment by simonguo** [comment, marked as answer]
Could you provide a codesandbox example or legend to illustrate your needs?

**Reply by cangSDARM** [reply]
https://codesandbox.io/s/agitated-vaughan-m07c4c?file=/src/App.js

**Reply by simonguo** [reply]
From your example I can't understand at all what you want?
And does it have anything to do with the distance to 38px、30px mentioned in your question?

**Reply by cangSDARM** [reply]
Simply in this example, I want change 1 into 2:

![image](https://user-images.githubusercontent.com/20441896/184114689-1ea7fc25-7edb-4298-9e60-97db3a1f46f3.png)
1

![image](https://user-images.githubusercontent.com/20441896/184114771-6ca5d0e5-94fb-4d4e-a9b3-6a62dfacc75d.png)
2

Notice that the 2nd Popover is closer to the div with the green background

**Reply by simonguo** [reply]
There is an 8px width reserved for the arrow icon, which you can completely override with `marginTop:0`.

```diff
<Popover
    ref={ref}
-   style={{ left, top }}
+   style={{ left, top, marginTop:0 }}
    {...whisperPassed}
    arrow={false}
    full
/>
```

**Reply by cangSDARM** [reply]
I think this is still a workaround, but it has solved my problem. Hope there will be a better solution in the future
